packages/sdk-codegen/src/codeGenerators.ts

```diff
@@ -88,7 +88,7 @@ export const Generators: Array<IGeneratorSpec> = [
   {
     factory: (api: ApiModel, versions?: IVersionInfo) =>
       new GoGen(api, versions),
-    language: 'Golang',
+    language: 'Go',
```

Comment:
These seem like potentially user facing fixes? Maybe consider `s/chore/fix/` in the PR title? But totally your discretion

Reply:
Sure, it can be a `fix`